Design/css/style.css

```diff
@@ -0,0 +1,357 @@
+  font-family: 'Cocogoose', sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+.campaign {
+  border: 5px solid #ec5242;
+}
+
+h1,
+h2 {
+  font-family: 'Cocogoose', sans-serif;
+}
+
+h3,
+h4 {
+  font-family: 'Lato', sans-serif;
+}
+
+p {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  font-weight: 400;
+}
+
+.intro > h3 {
+  font-size: 28px;
+  color: #ec5242;
+}
+
+.about_desc > p {
+  font-family: 'Lato', sans-serif;
+  font-size: 22px;
+  font-weight: 400;
+}
+
+.main_program {
+  background-image: url("../images/program-bg.jpg");
+  background-size: cover;
+}
+
+a {
+  cursor: pointer;
+}
+
+.nav-item > a {
+  color: #000;
+  text-decoration: none;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+}
+
+.campaign > a {
+  color: #ec5242;
+}
+
+.nav-item > a:hover {
+  color: #ec5242;
+}
+
+.orange {
+  color: #ec5242;
+}
+
+.row {
+  margin-left: 0;
+  margin-right: 0;
+}
+
+.section_title_hr {
+  border: 1px solid #ec5242;
+  width: 5%;
+}
+
+.section_speaker_hr {
+  border: 1px solid #ec5242;
+  width: 10%;
+}
+
+.hr-title {
+  border: 1px solid #ec5242;
+  width: 5%;
+}
+
+.section_title {
+  font-size: 24px;
+}
+
+.section-subtitle {
+  font-size: 24px;
+}
+
+.logo {
+  padding-top: 4px;
+  padding-bottom: 4px;
+}
+
+.logo img {
+  border: 2px solid #000;
+  border-radius: 8px;
+}
+
+.nav-small {
+  font-size: 3rem;
+  top: 15%;
+  left: 5%;
+}
+
+.slider-about {
+  background: #f8f8f8;
+  background-image: linear-gradient(rgba(248, 248, 248, 0.7), rgba(248, 248, 248, 1));
+  background-size: cover;
+}
+
+.bg-dark-light-custom {
+  background-color: rgba(143, 143, 143, 0.5);
+}
+
+.bg-dark-custom {
+  background-color: #272a31;
+}
+
+.intro_description {
+  background-color: #f7f7f7;
+  font-family: 'Lato', sans-serif;
+  border: 1px solid #fff;
+  margin-left: -15px;
+  color: black;
+  width: 40%;
+}
+
+.program-block:hover {
+  border: 2px solid #fff;
+}
+
+.speaker-avatar {
+  top: 10px;
+  left: 22px;
+  width: 100%;
+  height: 80%;
+  z-index: 2;
+  margin-top: -50px;
+}
+
+.img1 {
+  width: 80px;
+  height: 70px;
+  margin-left: -20px;
+}
+
+.partners-section {
+  background-color: #414246;
+}
+
+.partners {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+}
+
+.partners > li {
+  margin-right: 80px;
+  padding: 10px;
+}
+
+.partners > li > img {
+  width: 100px;
+}
+
+.footer_img {
+  width: 100px;
+}
+
+ul {
+  list-style: none;
+}
+
+.vanish {
+  display: none;
+}
+
+.blur {
+  filter: blur(10px);
+}
+
+.stop-scroll {
+  overflow-y: hidden;
+}
+
+.bg-orange {
+  background-color: #ec5242;
+}
+
+.prev_1 {
+  background-image: linear-gradient(rgba(236, 82, 66, 0.5), rgba(236, 82, 66, 0.5)), url("../images/Sw.jpg");
+  background-size: cover;
+}
+
+.prev_2 {
+  background-image: linear-gradient(rgba(236, 82, 66, 0.5), rgba(236, 82, 66, 0.5)), url("../images/Sw2.jpeg");
+  background-size: cover;
+}
```

Comment:
- [ ] [OPTIONAL] These images look good on mobile and medium-size desktop screens, but larger screens do not show the entire image. That is probably because of the `background-size: cover` property. Try to play around it a bit to find a better property that matches more screens, or even better, you can apply some media queries to handle the behavior of your images.
Here's an image so you can see what I mean:
![Screenshot_2022-02-09_19-10-58](https://user-images.githubusercontent.com/59100477/153263436-05aba994-e399-4762-a208-0a12682ab187.png)

Reply:
Ok i will adjust to contain. Thanks for suggestion.